site/content/user-guide/jmeter_load_tests.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "JMeter Load Tests"
+weight = 280
```

Comment:
this is the same weight as that of the `java_client_sender.md`. You need to set this to a value matching the (relative) position you want the page to occur at in the menu.